service/host.go

```diff
@@ -164,6 +164,7 @@ func (s *manager) shutdown(err error, reason string, exitCode *int) (bool, error
 	}
 
 	// Flush tracing buffers
+	// GFM-378 uhttp.Client is not safe to send requests anymore
```

Comment:
I opened it when starting working on moving tracer out, we close a jaeger reporter here and tracer will panic if any requests will be made by http client